packages/sw-build-webpack-plugin/index.js

```diff
@@ -0,0 +1,65 @@
+const swBuild = require('sw-build');
+const path = require('path');
+const SERVICE_WORKER_NAME = 'sw.js';
+const GLOB_PATTERNS = '**\/*.{html,js,css}';
```

Comment:
Why errors? If we are going to handle the defaults in `sw-build` I am not sure why is it a necessity for the dev to define these.